src/components/styles/_outline.less

```diff
@@ -40,6 +40,6 @@
 
   &.white-btn {
     border-color: @whiteBorder;
-    color: @darkText !important;
+    color: @whiteBorder !important;
```

Comment:
instead of using the same variable please assign a new variable as it makes it easier to change if someone wants to modify the colors. Also make sure to run `npm run compile` once you're done.

Reply:
@shahednasser  have a review